pkg/cmd/openshift-sdn-node/proxy.go

```diff
@@ -23,7 +24,7 @@ import (
 func readProxyConfig(filename string) (*kubeproxyconfig.KubeProxyConfiguration, error) {
 	o := kubeproxyoptions.NewOptions()
 	o.ConfigFile = filename
-	if err := o.Complete(); err != nil {
+	if err := o.Complete(new(pflag.FlagSet)); err != nil {
```

Comment:
you need to pass down `cmd.Flags()` from above

Reply:
let me know if what I did is not right